File: android/versioned-abis/expoview-abi38_0_0/src/main/java/abi38_0_0/expo/modules/localauthentication/LocalAuthenticationPackage.java
package abi38_0_0.expo.modules.localauthentication;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi38_0_0.org.unimodules.core.BasePackage;
import abi38_0_0.org.unimodules.core.ExportedModule;

public class LocalAuthenticationPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.<ExportedModule>singletonList(new LocalAuthenticationModule(context));
  }
}

File: android/versioned-abis/expoview-abi44_0_0/src/main/java/abi44_0_0/expo/modules/localauthentication/LocalAuthenticationPackage.kt
package abi44_0_0.expo.modules.localauthentication

import android.content.Context
import abi44_0_0.expo.modules.core.BasePackage
import abi44_0_0.expo.modules.core.ExportedModule

class LocalAuthenticationPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf<ExportedModule>(LocalAuthenticationModule(context))
  }
}
